### PR TITLE
Smarthelp Enhancement

### PR DESCRIPTION
**Pull request by shivareddy**

### CODE REVIEW

File: VizbeeSmartPromptOptions.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default class VizbeeSmartPromptOptions {
+    
+    constructor() {
+        this.enabledSubflows = 
+        SmartPromptSubflow.CastAuthorization | SmartPromptSubflow.CastIntroduction | SmartPromptSubflow.SmartInstall;
+    }
+}
+
+export const SmartPromptSubflow = {
```
Comment (naveenk1223) [inline VizbeeSmartPromptOptions.js:9]: Can you let me know how will the user use this to set the sub-flows?

Comment (shivareddy) [reply VizbeeSmartPromptOptions.js:9]:
```
import {VizbeeSmartPromptOptions, SmartPromptSubflow } from 'react-native-vizbee-sender-sdk';

const smartPromptOptions = new VizbeeSmartPromptOptions();
    smartPromptOptions.enabledSubflows = SmartPromptSubflow.CastIntroduction | SmartPromptSubflow.SmartInstall
    VizbeeManager.smartPromptWithOptions(smartPromptOptions);
```

File: VizbeeManager.js [hunk 2]
```diff
@@ -24,10 +25,15 @@ class VizbeeManager {
         VizbeeNativeManager.smartPrompt();
     }
 
+    smartPromptWithOptions(smartPromptOptions) {
+        console.log(`VizbeeSmartPromptOptions ${smartPromptOptions}`);
```
Comment (naveenk1223) [inline VizbeeManager.js:29]: Can you delete the console.log?

Comment (shivareddy) [reply VizbeeManager.js]: Done

**Review by naveenk1223 — APPROVED** [review]

**Pull request merged**